services/prepare-essentials

```diff
@@ -13,15 +13,22 @@ logger.level = process.env.LOG_LEVEL || 'info';
 
 yargs.command('propose-updates',
   'Fetch the latest Update Center and propose updates',
-  () => {},
+  {uc: {}},
   (argv) => {
-    const uc = UpdateCenter.fromFile('./update-center.json')
+    const uc = UpdateCenter.fromFile(argv.uc)
     const manifest = Manifest.loadFile();
-    manifest.getPlugins().forEach((plugin) => {
+    manifest.getActualPlugins().forEach((plugin) => {
```

Comment:
The updater was failing to show updates to what was actually being delivered in `status`! It only listed newer versions of plugins which were _already_ in `spec`. As I have noted before, this is not helpful, since for example `workflow-aggregator` itself does nothing and should have been removed as of Jenkins 2; we need to actually enumerate the plugins we want to ship.